web/routes/general.py

```diff
@@ -37,7 +37,18 @@ def monitor(device_id):
     :param device_id: Device ID
     :return: Rendered template
     """
-    return render_template("gpio/monitor.html",
+    return render_template("/gpio/monitor.html",
```

Comment:
This slash does not affect the routing, though it breaks IDE's template recognition. Consider removing it.